include/wx/module.h

```diff
@@ -77,7 +77,7 @@ class WXDLLIMPEXP_BASE wxModule : public wxObject
     // its own
     void AddDependency(const char *className)
     {
-        m_namedDependencies.Add(className);
+        m_namedDependencies.Add(wxString(className, wxConvLibc));
```

Comment:
Should be `wxASCII_STR()` too, I think. It surely doesn't make sense to use `wxConvLibc` (runtime option) for a string defined at compile-time inside the program.